src/components/posts/Facebook/socialMedia/AdDisplay.js

```diff
@@ -1,24 +1,61 @@
 import React, { useContext } from "react";
 import { SocialMediaContext } from "./socialMediaHandler";
+import adData from "../../../../assets/Facebook/adContent.js";
+import "./socialMedia.css";
 
 export default function AdDisplay(props) {
   const { selectedAd } = useContext(SocialMediaContext);
+
   if (selectedAd === "default")
     return (
-      <div id="shown-ad">
-        <p>
-          Imagine the user scrolling through their home feed and seeing a job
-          ad.
-        </p>
-        <p>
-          The histogram below displays the frequency of different ads the user
-          might see each time they refresh their feed.
+      <div>
+        <div className="iphone-top"></div>
+        <div className="iphone-screen">
+          <p>
+            Imagine the user scrolling through their home feed and seeing a job
+            ad.
+          </p>
+          <p>
+            The histogram below displays the frequency of different ads the user
+            might see each time they refresh their feed.
+          </p>
+          <p>
+            Simulate some refreshes, and then click on a bar to view the
+            corresponding ad!
+          </p>
+        </div>
+      </div>
+    );
+  return (
+    <div>
+      <div className="iphone-top"></div>
+      <div className="iphone-screen">
+        <img
+          src={adData[selectedAd].img}
+          alt={`${selectedAd} clipart`}
+          className="ad-image"
+        ></img>
+        <h3>{adData[selectedAd].header}</h3>
+        <p>{adData[selectedAd].subheader}</p>
+        <a
+          href={adData[selectedAd].link}
+          target="_blank"
+          rel="noopener noreferrer"
+          className="ad-apply-link"
+        >
+          Apply Now!
+        </a>
+        <div className="ad-section-bar"></div>
+        <p>{adData[selectedAd].summary}</p>
+        <p className="ad-specs">
+          <div class="text-bold ad-specs-title">Requirements: </div>
+          {adData[selectedAd].reqs}
         </p>
-        <p>
-          Simulate some refreshes, and then click on a bar to view the
-          corresponding ad!
+        <p className="ad-specs">
+          <div class="text-bold ad-specs-title">Pay: </div>
```

Comment:
Do you think that the pay and the requirements section should be aligned together? It looks a bit off to me as is.
<img width="300" alt="Screen Shot 2021-08-03 at 10 26 03 PM" src="https://user-images.githubusercontent.com/65370631/128126803-51d37900-e425-47bd-8ee7-4c8667bdc25f.png">